productsArray.js

```diff
@@ -0,0 +1,102 @@
+export const productsArray = [
+    {
+        id: 0,
+        name: 'R2D2 Bag',
+        image: 'https://github.com/alchemycodelab/fsjs-foundations-march-2020/blob/master/curriculum/class-11/lab/assets/bag.jpg?raw=true'  
```

Comment:
I think your images are taking so long to load because you're linking to the on GitHub instead of downloading them to the assets folder. I know you've been resistant to changing that approach :) but the user experience isn't great because the lag is pretty significant.